custom/icds/tasks/data_pulls.py

```diff
@@ -34,10 +37,23 @@ def run_data_pull(data_pull_slug, month, location_id=None, email=None):
         raise
     else:
         if email and filename:
+            db = get_blob_db()
+            download_id = DownloadBase.new_id_prefix + make_uuid()
+            with open(filename, 'rb') as _file:
+                db.put(
+                    _file,
+                    domain=domain,
+                    parent_id=domain,
+                    type_code=CODES.data_export,
+                    key=download_id,
+                    timeout=24 * 60,
```

Comment:
The units on this timeout are "minutes", which was a regrettable decision on my part—seconds would have been more standard even if it would be impractical for a blob to timeout in 1s. I'm guessing that's what caused you to make the change below (see my other comment).